recipes/fenics-ffc/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = "2019.1.0" %}
+
+package:
+  name: fenics-ffc
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/fenics-project/ffc/downloads/ffc-{{ version }}.tar.gz
+  sha256: 4ff821a234869d8b9aaf8c5d7f617d42f9c134a2529e76c9519b681dff35affd
+  patches:
+    - ufc-include-path.patch
+
+build:
+  number: 39
```

Comment:
0?

Reply:
As mentioned in the description, these packages used to be built as part of multiple outputs in fenics-feedstock, so they are continuing their build numbers as standalone feedstocks (with the exception of fenics-ufl-legacy, which hasn't yet been on conda-forge, so starts at 0.)

Reply:
I thought there was a name change along the way.  Hard to follow all the changes here. Sorry.

Reply:
Not a problem, it's a weird situation. Thanks for reviewing!